perf_dashboard/configurations/views.py

```diff
@@ -19,7 +19,7 @@
 # Create your views here.
 def configuration(request):
     # Parse data for the master
-    df = pd.read_csv("/Users/carolynprh/PycharmProjects/perf_dashboard/perf_data/tmp3f9jejbf.csv")
+    df = pd.read_csv("/Users/carolynprh/istio-all/tools/perf_dashboard/perf_data/master.20191203-00.e4175a0ebbfd51bfdd99b84efef75ceb44d26958.csv")
```

Comment:
why is there this stuff from your personal directory ? Please remove this.

Reply:
yes, Just ignore this. this is just a placeholder. next PR would be the configuration part.